File: app/src/main/java/io/github/domi04151309/alwayson/DialogAdmin.java
package io.github.domi04151309.alwayson;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.Button;

public class DialogAdmin extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.dialog_admin);

        Button close_button = findViewById(R.id.okay_btn);
        close_button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(DialogAdmin.this, Preferences.class));
                finish();
            }
        });
    }
}

File: app/src/main/java/io/github/domi04151309/alwayson/DialogAdmin.kt
package io.github.domi04151309.alwayson

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.widget.Button

class DialogAdmin : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.dialog_admin)

        val closeBtn = findViewById<Button>(R.id.okay_btn)
        closeBtn.setOnClickListener {
            startActivity(Intent(this@DialogAdmin, Preferences::class.java))
            finish()
        }
    }
}
